setup.py

```diff
@@ -1,16 +1,24 @@
-from distutils.core import setup
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
 setup(
     name="braintree",
     version="3.29.2",
     description="Braintree Python Library",
+    long_description="""
```

Comment:
We're adding `long_description` with some custom information in another commit, remove this change.